### ISSUE TITLE
Supabase new update with streaming, eq and gte errors in list views

### ISSUE THREAD

**Issue by jakals37** [FlutterFlow/flutterflow-issues#4893]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

When sending app to app store and upload on web, i get the following errors. All is related to equal and greater than equal in listviews with the supabase update:

Target dart2js failed: ProcessException: Process exited abnormally with exit code 1:
lib/a_tab_bars/events/events_widget.dart:212:50:
Error: The method 'gte' isn't defined for the class 'SupabaseStreamBuilder'.
 - 'SupabaseStreamBuilder' is from 'package:supabase/src/supabase_stream_builder.dart' ('/opt/.pub-cache/hosted/pub.dev/supabase-2.3.0/lib/src/supabase_stream_builder.dart').
                                                .gte(
                                                 ^^^
lib/a_tab_bars/events/events_widget.dart:303:52:
Error: The method 'gte' isn't defined for the class 'SupabaseStreamBuilder'.
 - 'SupabaseStreamBuilder' is from 'package:supabase/src/supabase_stream_builder.dart' ('/opt/.pub-cache/hosted/pub.dev/supabase-2.3.0/lib/src/supabase_stream_builder.dart').
                                                  .gte(
                                                   ^^^
lib/a_tab_bars/events/events_widget.dart:445:64:
Error: The method 'gte' isn't defined for the class 'SupabaseStreamBuilder'.
 - 'SupabaseStreamBuilder' is from 'package:supabase/src/supabase_stream_builder.dart' ('/opt/.pub-cache/hosted/pub.dev/supabase-2.3.0/lib/src/supabase_stream_builder.dart').
                                                              .gte(
                                                               ^^^
lib/a_tab_bars/home/home_widget.dart:162:46:
Error: The method 'eq' isn't defined for the class 'SupabaseStreamBuilder'.
 - 'SupabaseStreamBuilder' is from 'package:supabase/src/supabase_stream_builder.dart' ('/opt/.pub-cache/hosted/pub.dev/supabase-2.3.0/lib/src/supabase_stream_builder.dart').
                                            .eq(
                                             ^^
lib/a_tab_bars/home/home_widget.dart:895:56:
Error: The method 'eq' isn't defined for the class 'SupabaseStreamBuilder'.
 - 'SupabaseStreamBuilder' is from 'package:supabase/src/supabase_stream_builder.dart' ('/opt/.pub-cache/hosted/pub.dev/supabase-2.3.0/lib/src/supabase_stream_builder.dart').
                                                      .eq(
                                                       ^^
lib/more_screens/edit_group/edit_group_widget.dart:1051:36:
Error: The method 'eq' isn't defined for the class 'SupabaseStreamBuilder'.
 - 'SupabaseStreamBuilder' is from 'package:supabase/src/supabase_stream_builder.dart' ('/opt/.pub-cache/hosted/pub.dev/supabase-2.3.0/lib/src/supabase_stream_builder.dart').
                                  .eq(
                                   ^^
Error: Compilation failed.
#0      RunResult.throwException (package:flutter_tools/src/base/process.dart:122:5)

### Expected Behavior

For the apps to publish.

### Steps to Reproduce

1. Current project with supabase.
2. Publish to app store

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4wi8mAvI9IsbRJ+KqJccAxnjwWGDt+U7g7lNVDbCs8GqLuPe0UWMzBUBFUOem7dAlieFn9hnEX/MnIi9jXCPAEJTObQNl9zJZUazqXfF6gV7a2O4S3b0dSFplTIEis4bW7rhIlXctYSHw76meuHvO9SXbAO5jtIlU7POa4XdeK2SrDX1iXc2URm05KZDPz

### Visual documentation

![image](https://github.com/user-attachments/assets/94a18e12-074a-40fa-b0ce-585bbbb4f87f)
![image](https://github.com/user-attachments/assets/2478e144-1833-4841-a62d-31d79639dc5b)

### Environment

```markdown
- FlutterFlow version: FlutterFlow v5.0.11+ released November 13, 2024 Flutter version is 3.24.2
- Platform: Windows
- Browser name and version:ndows N/A
- Operating system and version affected: Windows 10
```

**Comment by jakals37** [comment]
It looks like its an issue when there are more than one equal statements in the code. 2nd one gives and error

**Comment by Alezanello** [comment]
Hello!

Thank you for bringing this to our attention. I can confirm this issue and will escalate it to the engineering team as a high priority for a fix.

**Comment by SlvrB** [comment]
I have a same issue

**Comment by darG37** [comment]
Same here, can't finish my project because of this bug.

**Comment by MrRongoose** [comment]
Following, same issues

**Comment by Asclepeion** [comment]
Following, got the same issue. It’s completely halted progress on my project.

**Comment by sabatage3** [comment]
I have that issue!! this is catastrophic!

**Comment by glgh** [comment]
Hey all - this is unfortunately a limitation of the Supabase streaming API, as it doesn't support multiple filters. To fix the error, please remove the extra filters, such that there are up to one filter; and we are going to remove the ability to set multiple queries in the next update.

Note that this is only a limitation on Supabase streaming queries - regular queries can continue to have multiple filters.

**Comment by ElnazWiseworks** [comment]
I got the same error!

**Comment by MrRongoose** [comment]
Does anyone know if you can use a single filter in the streaming query and add an additional filters via Generate Children below the stack?

**Comment by ryanmish** [comment]
@MrRongoose I was able to get this to work successfully (as far as I can tell). Did you try?

**Comment by MrRongoose** [comment]
> @MrRongoose I was able to get this to work successfully (as far as I can tell). Did you try?

@ryanmish Yes it works, I found that my issue was I was querying a view. I previously used a custom code with SupaFlow to subscribe to the main table with a callback action to refresh the view query. Since views do not broadcast realtime updates, it probably should not let you select it and turn off single time query (@glgh @leighajarett) My workaround was to make a column "auto-generate" on Supabase side, replicating the columns I had in the view in the main table. Basically the view was a merge of two tables so I set a trigger to update the column on insert or update and use a query on the main table in FlutterFlow
